packages/plugin-utils/src/options.ts

```diff
@@ -120,6 +120,11 @@ export interface UserOptions {
    */
   sortUtilities?: boolean
 
+  /**
+   * Callback when classes and/or tags are generated/changed
+   */
+  onGenerated?: (classes: Set<string>, tags: Set<string>) => void
```

Comment:
```suggestion
  onGenerated?: (ctx: { classes: Set<string>, tags: Set<string> }) => void
```

Would prefer passing as an object for extensibility in the future.